perses/app/fah_generator.py

```diff
@@ -237,9 +279,13 @@ def run_neq_fah_setup(ligand_file,
                       protein_kwargs=None,
                       ionic_strength=0.15*unit.molar,
                       remove_constraints=False,
-                      rmsd_restraint=True,
+                      rmsd_restraint=False,
+                      impose_virtual_bonds=True,
+                      use_given_geometries=True,
```

Comment:
Since we are now defaulting to `use_given_geometries=True` I think we have to change the test [here](https://github.com/choderalab/perses/blob/67b7a719dc385e6fe359a847d33d92c6264f38b6/perses/tests/test_fah.py#L64-L71) to use `use_given_geometries=False`, since the complex phase is not being simulated (not even needed since it's a small molecule test). Does this seem reasonable?